projects/prison-identifier-and-delius/deploy/database/access.yml

```diff
@@ -2,8 +2,10 @@ database:
   access:
     username_key: /prison-identifier-and-delius/db-username
     password_key: /prison-identifier-and-delius/db-password
+  tables:
+    - offender
```

Comment:
it wasn't mentioned in the ticket but I can add them... @anthony-britton-moj @mxco86 Any thoughts?

Reply:
There is this ticket: https://dsdmoj.atlassian.net/browse/PI-1643 which talks about key dates but this is just for a report rather than actually correcting the data.

Reply:
Actually we need the booking number for key dates don't we?